src/css/wui.css

```diff
@@ -1,4 +1,26 @@
+.absolute-left {
+    position: absolute;
+    left: 5px;
+    top: 2px;
+}
 
+.left {
```

Comment:
This rule is a non-semantic sub-atomic particle where the rule and the class name are redundant.